brands/amenity/bank.json

```diff
@@ -52,7 +58,8 @@
       "brand": "ANZ",
       "brand:wikidata": "Q714641",
       "brand:wikipedia": "en:Australia and New Zealand Banking Group",
-      "name": "ANZ"
+      "name": "ANZ",
+      "official_name": "Australia and New Zealand Banking Group Limited"
```

Comment:
Again - is it really official bank name? Not the official name of owner?

Reply:
I don't know. Wikipedia says "The Australia and New Zealand Banking Group Limited, commonly called ANZ, is the third largest bank by market capitalisation in Australia."

Reply:
OK, remove the other one (Absa Group Limited) and I will merge.